locales/en-US/editor.properties

```diff
@@ -145,7 +145,8 @@ DND_SUCCESS_UNZIP=Successfully unzipped <b>{0}</b>.
 DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 
 # Image Viewer
-IMAGE_DIMENSIONS={0} (width) &times; {1} (height) pixels
```

Comment:
All these string changes need to happen in Thimble vs. here, can you open a PR to do it there vs. here?  Also, @gideonthomas, what's the process for removing strings here?  Do we do it manually?